src/components/Account/Avatar.js

```diff
@@ -77,11 +87,11 @@ const Avatar = props => {
 };
 
 Avatar.propTypes = {
-  avatarURL: PropTypes.string
+  avatarURL: PropTypes.string,
```

Comment:
@Envl it looks like your Prettier configuration has changed.
Throughout the code base, I have disabled trailing commas after the last item of a list or hashmap.